src/commands/flex/plugins/deploy.ts

```diff
@@ -82,6 +84,10 @@ export default class FlexPluginsDeploy extends FlexPlugin {
    * @override
    */
   async doRun() {
+    if (this.builderVersion !== 4) {
```

Comment:
i thought deploy works with v3? 🤔

Reply:
Not the full feature - We have an older version of deploy with the CLI, but if you upgrade to this new version, you need to update your plugin script as well.